wordpress-mobile/WordPress-Android
# Update fastlane and release-toolkit

GitHub claims a `potential security vulnerability` warning on the repository due to `nokogiri`. 
This PR updates `release-toolkit` to the latest version in order to use a newer version of `nokogiri`. The PR also updates `fastlane` to the latest. 

To test:
 - Check that CI is green. 
 - Check that `bundle install` succeeds locally.
 
PR submission checklist:
